tests/form-component.spec.ts

```diff
@@ -1448,4 +1448,27 @@ test.describe('Form Component', () => {
       expect(dump.form).toEqual({ child: 'A' })
     })
   })
+
+  test.describe('getData and getFormData methods', () => {
+    test.beforeEach(async ({ page }) => {
+      consoleMessages.listen(page)
+      await page.goto('/form-component/data-methods')
+    })
+
+    test('getData returns form data as object', async ({ page }) => {
+      await page.fill('#name', 'John Doe')
+      await page.getByRole('button', { name: 'Test getData()' }).click()
+
+      const result = consoleMessages.messages.find((msg) => msg.includes('getData result:'))
```

Comment:
Okay, `consoleMessages` was the missing piece. I didn't know about this and that I could listen to `console.log()` this way.